core/src/serve_repair.rs

```diff
@@ -755,7 +838,7 @@ impl ServeRepair {
         let mut pending_pings = Vec::default();
 
         // iter over the packets
-        for (i, packet) in packet_batch.iter().enumerate() {
+        for packet in packet_batch.iter() {
```

Comment:
I think we probably want to respond to requests in order of their stake.
Otherwise what if all the unstaked requests become first or this is overwhelmed with so many unstaked requests.